spec/models/user_spec.rb

```diff
@@ -0,0 +1,51 @@
+require "rails_helper"
+
+describe User do
+  before do
+    User.delete_all
+  end
+
+  describe ".ordered" do
+    it "orders records by name ASC" do
+      beatrice = User.create!(name: "Beatrice", email: "beatrice@interexchange.org", password: "password")
+      adam = User.create!(name: "Adam", email: "adam@interexchange.org", password: "password")
+
+      expect(described_class.ordered).to eq([adam, beatrice])
+    end
+  end
+
+  describe ".search" do
+    subject { described_class.search(search_term) }
+
+    let!(:david) { User.create!(name: "David", email: "david@interexchange.org", password: "password") }
+    let!(:candice) { User.create!(name: "Candice", email: "candice@interexchange.org", password: "password") }
+
+    describe "searching by name" do
+      let(:search_term) { "cand" }
```

Comment:
This matches both on name and email.

You should change your sample data so this search term doesn't apply to both, thus proving that we can actually search on name.

Right now if you removed the name clause from your search I believe this would still pass.